docs/pages/access-controls/access-requests.mdx

```diff
@@ -11,17 +11,20 @@ denied based on a configurable number of approvers.
 Just-in-Time Access Requests are a powerful way to implement the principle of
 least privilege in your organization, leaving an attacker with no permanent
 admins to target. Users receive elevated privileges for a limited period of
-time. And aside from their reviewer privileges, users who review requests can
-have limited access to cluster resources.
+time. Request approvers can be configured with limited cluster access so they
+are not high value targets.
 
 <ScopedBlock scope={["oss"]}>
```

Comment:
I added a commit adding the scope selectors to this page, but in general I don't see the advantage of hiding content behind scoped blocks to an small informational page like this. I suggest removing them.